### ISSUE TITLE
Is it possible to set the DEPLOY_MODE to cluster?

### ISSUE THREAD

**Issue by seybi87** [ywilkof/spark-jobs-rest-client#15]
Hi, 

I did some experimental deployments with this useful REST client.

I am wondering if it is possible to set DEPLOY_MODE to cluster as the default value is client?

I've seen there is the DEPLOY_MODE_CLUSTER variable in the code, which seems not be used.

Thanks!

**Comment by ywilkof** [comment]
You're right, it is not being used. Why should we supply deploy mode? It seems that since it was never given it does not have any effect. When using the REST API, the driver starts on one of the nodes, which is in line with cluster deploy mode as a matter of fact, not with client deploy mode. I'm not sure then that client mode is the default when using the REST endpoints. 
Maybe we should just remove the unused variable?
